packages/components/src/components/domainproperties/DomainRowExpandedOptions.tsx

```diff
@@ -92,7 +93,7 @@ export class DomainRowExpandedOptions extends React.Component<IDomainRowExpanded
                     <Col xs={12} lg={10}>
                         <NameAndLinkingOptions index={index} field={field} onChange={onChange}/>
                     </Col>
-                    {!isFieldFullyLocked(field.lockType) &&
+                    {!appPropertiesOnly && !isFieldFullyLocked(field.lockType) &&
```

Comment:
As discussed, change this to only hide the Conditional Format Criteria and leave the regex and range validator options.